PhotoLocator/AutoTagViewModel.cs

```diff
@@ -139,15 +139,29 @@ protected bool SetProperty<T>(ref T field, T newValue, [CallerMemberName] string
         
         private IEnumerable<GpsTrace> LoadAdditionalGpsTraces()
         {
-            if (string.IsNullOrEmpty(TraceFilePath))
+            if (string.IsNullOrWhiteSpace(TraceFilePath))
                 return GpsTraces;
+
+            // Support Windows Copy Full Path string to be used by the user, Example: "C:\temp\file.jpeg"
+            if (TraceFilePath.Contains("\"")) TraceFilePath = TraceFilePath.Replace("\"", "");
```

Comment:
I suggest using `TraceFilePath.Trim(' ', '"')` instead, it will also handle leading and trailing space

Reply:
I never saw the use of Trim() for this. It looks really neat. I haven't tested this, but using `Trim(' ', '"') will also trim whitespaces?`

Reply:
Yes, this will trim both " and whitespaces.